Sources/Xcodeproj/pbxproj().swift

```diff
@@ -351,6 +351,7 @@ func xcodeProject(
                 targetSettings.common.PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)"
                 targetSettings.common.PRODUCT_MODULE_NAME = "$(TARGET_NAME:c99extidentifier)"
                 targetSettings.common.PRODUCT_BUNDLE_IDENTIFIER = module.c99name.mangledToBundleIdentifier()
+                targetSettings.common.SKIP_INSTALL = "YES"
```

Comment:
`SKIP_INSTALL` is not configured on executable target in Xcode's default project template.